source/mus_fluid_module.f90

```diff
@@ -592,16 +593,18 @@ subroutine mus_fluid_dump( me, minLevel, maxLevel, physics, nSolve, general, &
       do iLevel = 1, 3
         write(outUnit, "(A,I2,A,F10.7)") '  ', iLevel,' = ', me%omega_Lim(iLevel)
       enddo
-      
+
       ! check Cumulant_extended range of admissibility for omegas
-      write(logUnit(1),*) 'Checking admissibility of omegas for parametrized Cumulant scheme'
+      write(logUnit(1),*) 'Checking admissibility of omegas for parametrized &
+        &                  Cumulant scheme'
```

Comment:
Could we avoid using strings over two lines? Instead of doing that, I'd prefer to have:

```suggestion
      write(logUnit(1),*) 'Checking admissibility of omegas for parametrized' &
        &                 // ' Cumulant scheme'
```